extensions/azdata/package.nls.json

```diff
@@ -2,5 +2,23 @@
 	"azdata.displayName": "Azure Data CLI",
 	"azdata.description": "Support for Azure Data CLI.",
 	"azdata.config.title": "Azure Data CLI Configuration",
-	"azdata.config.debug": "Log debug info to the output channel for all executed azdata commands"
+	"azdata.config.debug": "Log debug info to the output channel for all executed azdata commands",
+
+	"command.category": "Azure Data CLI",
+	"azdata.acceptEula.command.name": "Accept Eula",
+	"azdata.install.command.name": "Install",
+	"azdata.upgrade.command.name": "Check for Upgrade",
+	"azdata.category": "Azure Data CLI",
+
+	"azdata.acceptEula.description": "Choose how acceptance of EULA for the Azure Data CLI is done",
+	"azdata.acceptEula.prompt.description": "The user will be prompted for acceptance of EULA for the Azure Data CLI",
+	"azdata.acceptEula.dontPrompt.description": "The user will not be prompted for acceptance of EULA for the Azure Data CLI",
+
+	"azdata.install.description": "Choose how install of Azure Data CLI is done",
+	"azdata.install.prompt.description": "The user will be prompted for installation of the Azure Data CLI",
+	"azdata.install.dontPrompt.description": "The user will not be prompted for installation of the Azure Data CLI",
+
+	"azdata.upgrade.description": "Choose how upgrade of Azure Data CLI tool is done",
+	"azdata.upgrade.prompt.description": "The user will be prompted for upgrade of the Azure Data CLI",
```

Comment:
Please use update instead of upgrade. Upgrade isn't really a term usually used for this type of thing

Reply:
And I'd prefer that you update the name across all the functions/variables too just to be consistent